Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -2,9 +2,45 @@ import macro from 'vtk.js/Sources/macros';
 import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
 import Constants from 'vtk.js/Sources/Rendering/Core/VolumeMapper/Constants';
 import vtkAbstractMapper from 'vtk.js/Sources/Rendering/Core/AbstractMapper';
+import vtkPiecewiseFunction from 'vtk.js/Sources/Common/DataModel/PiecewiseFunction';
+import vtkColorTransferFunction from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction';
 
 const { BlendMode, FilterMode } = Constants;
 
+function createRadonTransferFunctions(values) {
+  const ofun = vtkPiecewiseFunction.newInstance();
+  ofun.addPoint(-3024, 0.1);
+  ofun.addPoint(-637.62, 0.1);
+  ofun.addPoint(700, 0.5);
+  ofun.addPoint(3071, 0.9);
+
+  const ctfun = vtkColorTransferFunction.newInstance();
+  ctfun.addRGBPoint(-3024, 0.0, 0.0, 0.0);
+  ctfun.addRGBPoint(
+    values.min,
+    values.minValue,
+    values.minValue,
+    values.minValue
+  );
+  ctfun.addRGBPoint(
+    values.max,
+    values.maxValue,
+    values.maxValue,
+    values.maxValue
+  );
+  ctfun.addRGBPoint(3071, values.maxValue, values.maxValue, values.maxValue);
```

Comment:
Here in red is the look of your grayscale color transfer function (where `M1 = min(values.min, values.max)` and `M2 = max(values.min, values.max)`) :
![image](https://user-images.githubusercontent.com/219044/184343248-977c62df-1761-44c6-ab35-e6c93d2981f2.png)

Don't you want the blue line instead ?

Reply:
After testing the blue line, it doesn't seem to have any difference. But if you prefer, I can setup the blue line.